examples/oauth_producer.py

```diff
@@ -131,5 +124,8 @@ def main(args):
                         help="Token URL.")
     parser.add_argument('--scopes', dest="scopes", required=True, nargs='+',
                         help="Scopes requested from OAuth server.")
-
+    parser.add_argument('--logical-cluster', dest="logical_cluster", required=False, help="Logical Cluster.")
```

Comment:
If we move this to CC specific example, does it make sense to mark these are required?

Reply:
The original example is using an outdated callback based method so this example applies there too. Also, since the only difference between CC and AK/CP is this one param, it doesn't make sense to split it into another file. There's a wider task of updating all the examples to be using the recommended approach. 

The purpose of examples is to educate.

Reply:
We can improve old example as well and create a new example to cater mainly CC usecase. The idea is to make it easy for non CC user to use SASL as well. With everything specific to CC, user might get confused.

Reply:
It's not just specific to CC though. If you don't include the logicalClusterId and identityPoolId in the cli params, it's just just works against any AK or CP.

Reply:
I just don't want code that'll be copy pasted for a 2 line difference. The cli works with both CP and CC.